Source/Renderer/VertexArray.js

```diff
@@ -636,9 +673,52 @@ define([
         return this._attributes[index];
     };
 
+    // Work around for ANGLE, where the attribute divisor seems to be part of the global state instead
+    // of the VAO state. This function is called when the vao is bound, and should be removed
+    // once the ANGLE issue is resolved.
+    var divisors = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]; // at most 16 vertex attributes
+    var hadInstancedAttributes = false;
```

Comment:
Same comment.  This should be tracked per `Context` AFAIK like all the other GL state Cesium maintains.